packages/medusa/src/api-v2/auth/[scope]/[authProvider]/route.ts

```diff
@@ -0,0 +1,37 @@
+import { AuthenticationInput, IAuthModuleService } from "@medusajs/types"
+import { MedusaRequest, MedusaResponse } from "../../../../types/routing"
+
+import { MedusaError } from "@medusajs/utils"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+
+export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
+  const { scope, authProvider } = req.params
```

Comment:
suggestion: Maybe a middleware to validate the scope and provider so that the route is not open ended?